jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -228,9 +228,56 @@ YUI.add('user-profile', function() {
     switchEnv: function(uuid, name) {
       var props = this.props;
       props.showConnectingMask();
+      this.close();
       props.switchModel(uuid, this.state.envList);
       props.dbEnvironmentSet('name', name);
+    },
+
+    /**
+      Set to a new, blank canvas with no model.
+
+      @method resetModel
+    */
+    resetModel: function() {
```

Comment:
I wonder if this is something that should be done outside of the user profile, might save some code duplication later (e.g. shouldn't the new button in the model switcher do the same thing?).